resolver/dns.go

```diff
@@ -0,0 +1,33 @@
+package resolver
+
+import (
+	"fmt"
+	"net"
+)
+
+const (
+	testnetDomain = ".containers.testnet.fs.neo.org"
+	mainnetDomain = ".containers.fs.neo.org"
+)
+
+// ResolveContainerTestnet request txt record name + '.containers.testnet.fs.neo.org' to default dns server.
+func ResolveContainerTestnet(name string) (string, error) {
+	return resolveContainerName(name + testnetDomain)
+}
+
+// ResolveContainerMainnet request txt record name + '.containers.fs.neo.org' to default dns server.
+func ResolveContainerMainnet(name string) (string, error) {
+	return resolveContainerName(name + mainnetDomain)
+}
+
+func resolveContainerName(name string) (string, error) {
```

Comment:
Let's make this function public for https://github.com/nspcc-dev/neofs-s3-gw/issues/281
Private installation might use custom mapped domains.